pkgs/development/tools/gvls/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, glib
+, vala
+, jsonrpc-glib
+, json-glib
+, gtk3
+, pango
+, gtksourceview3
+, libgee
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gvls";
+  version = "0.14.3";
```

Comment:
Should be updated to 0.16.0: https://gitlab.gnome.org/esodan/gvls/-/tags

Reply:
Yep, was planned.